pkgs/applications/misc/pdfstudio/default.nix

```diff
@@ -31,27 +31,23 @@ in
 {
   pdfstudioviewer = callPackage ./common.nix rec {
     inherit desktopName pname program year;
-    version = "${year}.2.4";
+    version = "${year}.0.3";
     longDescription = ''
       PDF Studio Viewer is an easy to use, full-featured PDF editing software. This is the free edition. For the standard/pro edition, see the package pdfstudio.
     '';
     src = fetchurl {
       url = "https://download.qoppa.com/pdfstudioviewer/PDFStudioViewer_linux64.deb";
```

Comment:
Hey I just noticed that this links also works:

https://download.qoppa.com/pdfstudio/v2023/PDFStudio_v2023_0_3_linux64.deb

Meaning you can use the same URL schema as the other versions.

Reply:
The link you gave is the one for the full version, packaged as `pdfstudio2023`, which needs payment and activation. The package `pdfstudioviewer` is intended for the free version. I don't know of any link for the pdfstudioviewer deb file that matches the URL schema of the full versions.